bricksrc/quantities.py

```diff
@@ -1,109 +1,565 @@
             },
         },
     },
     "Voltage": {
-        "subclasses": {
-            "Electric_Voltage": {
-                "subclasses": {
-                    "Voltage_Magnitude": {},
-                    "Voltage_Angle": {},
-                    "Voltage_Imbalance": {},
-                },
+        "subconcepts": {
+            "Voltage_Magnitude": {OWL.sameAs: QUDTQK["Voltage"]},
+            "Voltage_Angle": {
+                SKOS.definition: Literal(
+                    "The angle of the phasor representing a measurement of electric voltage"
+                ),
+                QUDT.applicableUnit: [
+                    UNIT.ARCMIN,
+                    UNIT.ARCSEC,
+                    UNIT.DEG,
+                    UNIT.GON,
+                    UNIT.GRAD,
+                    UNIT.MIL,
+                    UNIT.RAD,
+                    UNIT.MicroRAD,
+                    UNIT.MilliRAD,
+                    UNIT.MilliARCSEC,
+                    UNIT.REV,
+                ],
+                QUDT.hasDimensionVector: QUDTDV["A0E0L0I0M0H0T0D1"],
+                SKOS.definition: Literal("Angle of voltage phasor"),
+                RDFS.isDefinedBy: URIRef(str(BRICK).strip("#")),
+                RDFS.label: Literal("VoltageAngle"),
+                SKOS.broader: BRICK.PhasorAngle,
+            },
+            "Voltage_Imbalance": {
+                SKOS.definition: Literal("The percent deviation from average voltage"),
+                QUDT.applicableUnit: [UNIT.PERCENT],
+                QUDT.hasDimensionVector: QUDTDV["A0E0L0I0M0H0T0D1"],
+                RDFS.isDefinedBy: URIRef(str(BRICK).strip("#")),
+                RDFS.label: Literal("VoltageImbalance"),
+                SKOS.broader: BRICK.Dimensionless,
+            },
+        },
+    },
+    "Dewpoint": {OWL.sameAs: QUDTQK["DewPointTemperature"]},
+    "Direction": {
+        "subconcepts": {
+            "Wind_Direction": {
+                QUDT.applicableUnit: [
+                    UNIT.ARCMIN,
+                    UNIT.ARCSEC,
+                    UNIT.DEG,
+                    UNIT.GON,
+                    UNIT.GRAD,
+                    UNIT.MIL,
+                    UNIT.RAD,
+                    UNIT.MicroRAD,
+                    UNIT.MilliRAD,
+                    UNIT.MilliARCSEC,
+                    UNIT.REV,
+                ],
+                QUDT.hasDimensionVector: QUDTDV["A0E0L0I0M0H0T0D1"],
+                SKOS.definition: Literal("Direction of wind relative to North"),
+                RDFS.isDefinedBy: URIRef(str(BRICK).strip("#")),
+                RDFS.label: Literal("Wind_Direction"),
+            }
+        }
+    },
+    "Energy": {
+        "subconcepts": {
+            "Electric_Energy": {OWL.sameAs: QUDTQK["Energy"]},
+            "Thermal_Energy": {OWL.sameAs: QUDTQK["ThermalEnergy"]},
+        },
+    },
+    "Flow": {
+        OWL.sameAs: QUDTQK["VolumeFlowRate"],
+        "subconcepts": {
+            "Flow_Loss": {
```

Comment:
Flow_Loss is not a subconcept of Flow.

Reply:
Why not? It's a "flavor" of flow that refers to the amount of flow that is lost rather than the amount of flow that is delivered

Reply:
This has been confirmed by the JCI folks: Flow Loss is also measured as volume flow rate